source/search_page/assets/script.js

```diff
@@ -0,0 +1,26 @@
+window.addEventListener('DOMContentLoaded', init);
+
+async function init() {
+    createRecipeCards();
+  }
+
+function createRecipeCards() {
+
+  
+    for(let i = 0; i<25;i++){
+
+    let ele = document.createElement('search-recipe');
+    if(i%2){
+      ele.data = "Delish Pasta";
+    }else{
+      ele.data = "Delish delish Pasta with bacon and even more pasta";
+    }
+    
+    
+    document.querySelector('main').append(ele)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8414003)
```suggestion
    document.querySelector("main").append(ele)
```